src/pages/workspaces/workspace/analysis/runtime-utils.js

```diff
@@ -0,0 +1,316 @@
+import _ from 'lodash/fp'
+import {
+  cloudServices, gpuTypes, machineTypes, zonesToGpus
+} from 'src/data/gce-machines'
+import * as Utils from 'src/libs/utils'
+import { allAppTypes, appTools, runtimeToolLabels } from 'src/pages/workspaces/workspace/analysis/utils/tool-utils'
+
+
+export const runtimeTypes = {
+  gceVm: 'Standard VM',
+  dataprocSingleNode: 'Spark single node',
+  dataprocCluster: 'Spark cluster'
+}
+
+export const pdTypes = {
```

Comment:
why are we redefining a model defined in `disk-models.ts` here in a js file dealing with runtimes?